src/event.js

```diff
@@ -28,6 +28,14 @@ module.exports.setException = function setException(event, error) {
     }
 };
 
+/**
+ * Add timeout indication to a given event
+ * @param {proto.event_pb.Event} event The event the timeout is set on
+ */
+module.exports.setTimeout = function setTimeout(event) {
```

Comment:
maybe markAsTimeout? (setTimeout is kinda reserved in node)

Reply:
K